winch/codegen/src/isa/aarch64/regs.rs

```diff
@@ -26,6 +27,7 @@ pub(crate) const fn ip0() -> Reg {
     xreg(16)
 }
 
+// TODO: delete this.
 /// Alias to the IP0 register.
 pub(crate) const fn scratch() -> Reg {
```

Comment:
Would it make sense to tag this as `#[deprecated]` and annotate the remaining uses with `#[expect(deprecated, reason = "...")]`?